draft-ietf-jsonpath-base.md

```diff
@@ -188,8 +188,7 @@ Name:
   It is included here for completeness.
 
 Element:
-: A value in an array.  (Also used with a distinct meaning in XML
-  context for XML elements.)
+: A value in an array.
```

Comment:
I'd probably keep a 

(not to be confused with XML element).

Reply:
Ok, I grudgingly agree. ;-) I promise a beer to the first person who reads the spec and says "What's XML?"...